.gitattributes

```diff
@@ -2,3 +2,4 @@
 *.erb eol=lf
 *.pp eol=lf
 *.sh eol=lf
+*.epp eol=lf
```

Comment:
we should probably add that to modulesync_config?

Reply:
This is PDK